lib/init.php

```diff
@@ -101,50 +101,3 @@ function gutenberg_rest_nonce() {
 	exit( wp_create_nonce( 'wp_rest' ) );
 }
 add_action( 'wp_ajax_gutenberg_rest_nonce', 'gutenberg_rest_nonce' );
-
-
-/**
- * Exposes the site icon url to the Gutenberg editor through the WordPress REST
- * API. The site icon url should instead be fetched from the wp/v2/settings
- * endpoint when https://github.com/WordPress/gutenberg/pull/19967 is complete.
- *
- * @since 8.2.1
- *
- * @param WP_REST_Response $response Response data served by the WordPress REST index endpoint.
- * @return WP_REST_Response
- */
-function register_site_icon_url( $response ) {
-	$data                  = $response->data;
-	$data['site_icon_url'] = get_site_icon_url();
-	$response->set_data( $data );
-	return $response;
-}
-
-add_filter( 'rest_index', 'register_site_icon_url' );
```

Comment:
@jasmussen, great catch. I accidentaly removed this filter related to the Site Logo block 😱

Reply:
Ready in #40041.